components/job-orchestration/job_orchestration/executor/compress/celery_compress.py

```
@@ -4,16 +4,19 @@
 
 from job_orchestration.executor.compress.celery import app
 from job_orchestration.executor.compress.compression_task import compression_entry_point
+from job_orchestration.executor.telemetry_utils import init_worker_telemetry
 
 # Setup logging
 logger = get_task_logger(__name__)
 
 
+@signals.worker_process_init.connect
+def worker_process_init_handler(**kwargs):
```

Ruff (0.15.15)

[warning] Missing type annotation for `**kwargs`

(ANN003)